configs/era5_example_config_preproc.cfg

```diff
@@ -20,7 +20,7 @@ partition_keys=
     month
     day
 [selection]
-product_type=ensemble_mean
+product_type=reanalysis
```

Comment:
Could we leave one as ensemble_mean? Or even make one ensemble_spread or ensemble_members? Just to cover more options in the examples.

Reply:
I'm ok with that. Will update now.